WORKSPACE

```diff
@@ -0,0 +1,38 @@
+git_repository(
+    name = "io_bazel_rules_go",
+    commit = "e0b19317b39357823b26c7e266596c8066e8f8e6",
+    remote = "https://github.com/bazelbuild/rules_go.git",
+)
+
+load("@io_bazel_rules_go//go:def.bzl", "go_repositories")
+
+go_repositories()
+
+debs = (
```

Comment:
They are for building docker base images. They are used in the docker packaging rules.